.github/workflows/build_docker.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build docker
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPO_NAME }}:latest
```

Comment:
In addition to `latest` we should have tag based on image commit hash: https://github.com/docker/build-push-action#customizing

Reply:
I'm not sure can we use `steps.commit.outputs.short` to get short commit hash? Done.

Reply:
I think we should use long commit is better.